profiles/common/modules/features/ec_embedded_video/themes/media-dailymotion-video.tpl.php

```diff
@@ -11,13 +11,19 @@
  *  $autoplay - If TRUE, then start the player automatically when displaying.
  *  $fullscreen - Whether to allow fullscreen playback.
  *  $output - The object/embed code.
+ *  $no_wrapper - Flag deterining if the video related tags must be embedded in
+ *  the container.
  *
  * Note that we set the width & height of the outer wrapper manually so that
  * the JS will respect that when resizing later.
  */
 ?>
-<div class="media-dailymotion-outer-wrapper" id="media-dailymotion-<?php print $id; ?>" style="width: <?php print $width; ?>px; height: <?php print $height; ?>px;">
+<?php if (!$no_wrapper): ?>
+  <div class="media-dailymotion-outer-wrapper" id="media-dailymotion-<?php print $id; ?>" style="width: <?php print $width; ?>px; height: <?php print $height; ?>px;">
```

Comment:
Shouldn't we add:

`class="<?php print $classes; ?> media-dailymotion-outer-wrapper"`

Like in other templates ?